mysql/mysqldump.go

```diff
@@ -238,6 +246,26 @@ func toSchemaKeys(columns []*ast.IndexPartSpecification) (keys []schema.Key) {
 	return keys
 }
 
+// toForeignKeys converts a string list of MySQL foreign keys to
+// schema foreign keys.
+func toForeignKeys(fk *ast.Constraint) (fkey schema.ForeignKey) {
+	columns := fk.Keys
+	referTable, _ := getTableName(fk.Refer.Table)
+	referColumns := fk.Refer.IndexPartSpecifications
+	var colNames, referColNames []string
+	for i, column := range columns {
+		colNames = append(colNames, column.Column.Name.String())
+		referColNames = append(referColNames, referColumns[i].Column.Name.String())
+	}
+	fkey = schema.ForeignKey{Columns: colNames,
+		Name:         fk.Name,
+		ReferTable:   referTable,
+		ReferColumns: referColNames,
+		OnDelete:     fk.Refer.OnDelete.ReferOpt.String(),
+		OnUpdate:     fk.Refer.OnUpdate.ReferOpt.String()}
+	return fkey
+}
+
 func updateCols(conv *internal.Conv, ct ast.ConstraintType, colNames []*ast.IndexPartSpecification, colDef map[string]schema.Column, tableName string) {
```

Comment:
updateCols probably no longer needs to set cd.Ignored.ForeignKey?
We're now mapping foreign key constraints into schema constraints, so no longer ignored.

Reply:
changed the behaviour, no longer ignoring foreign keys